app/models/ownership.rb

```diff
@@ -0,0 +1,6 @@
+class Ownership < ActiveRecord::Base
+	belongs_to :project
```

Comment:
so `attr_accessible #none` fixed that issue. but even without that line of code mass assignment is not possible as I tried in the console